specifyweb/frontend/js_src/lib/components/Header/NotificationRenderers.tsx

```diff
@@ -128,15 +130,17 @@ export const notificationRenderers: IR<
     const tableName = notification.payload.table;
     const collectionId = parseInt(notification.payload.collection_id);
     const mergeName = notification.payload.name;
-    const schemaResource = new schema.models.Collection.Resource({
-      id: collectionId,
-    });
-    const [collectionName, setCollectionName] = React.useState<string | null>(
-      ''
+    const collection = userInformation.availableCollections.find(
+      ({ id }) => id === collectionId
     );
-    schemaResource
-      .fetch()
-      .then((resource) => setCollectionName(resource.get('collectionName')));
+    const [collectionName, setCollectionName] = React.useState<
+      string | null | undefined
+    >('');
+
+    React.useEffect(() => {
+      setCollectionName(collection?.collectionName);
+    }, [collection]);
+
```

Comment:
why so complex?

```suggestion
const collectionName = collection?.collectionName;
```

and then you don't even need to create a variable and can use `collection?.collectionName` directly in the `return ...`